### PR TITLE
[LED-58, 59, 61] Adding UI functionalities

### PR DESCRIPTION
**Pull request by LadyMopar**
Added ability to insert a single transaction through the UI, choose a database file, add an account, and select an import file.

### CODE REVIEW

File: src/main/java/ledger/database/enity/AccountBalance.java [hunk 1]
```diff
@@ -21,17 +21,8 @@ public AccountBalance(int accountId, Date date, int amount) {
...
-    }
-
-    /**
-     * Sets the AccountBalance ID
-     *
-     * @param accountId The new id
-     */
-    public void setAccountId(int accountId) {
-        this.accountId = accountId;
+    public Account getAccount() {
```
Comment (shellajt) [inline src/main/java/ledger/database/enity/AccountBalance.java]: Did we lose the javadoc here?

File: src/main/java/ledger/user_interface/ui_controllers/AccountPopupController.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+import ledger.database.enity.AccountBalance;
+
+import java.net.URL;
+import java.util.Date;
+import java.util.ResourceBundle;
+
+/**
+ * Created by richarel on 10/16/2016.
+ */
+public class AccountPopupController extends GridPane implements Initializable {
```
Comment (shellajt) [inline src/main/java/ledger/user_interface/ui_controllers/AccountPopupController.java]: Can we put some documentation on the class?

File: src/main/java/ledger/user_interface/ui_controllers/AccountPopupController.java [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+            loader.load();
+        } catch (Exception e) {
+            System.out.println("Error on account popup startup: " + e);
+        }
+    }
+
+    /**
+     * Sets up action listener for the button on the page
+     *
+     * @param fxmlFileLocation
```
Comment (shellajt) [inline src/main/java/ledger/user_interface/ui_controllers/AccountPopupController.java]: Can we finish the param documentation?

File: src/main/java/ledger/user_interface/ui_controllers/LoginPageController.java [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    private TextField password;
+    @FXML
+    private Button loginBtn;
+
+    private String pwd;
+    private File file;
+
+    LoginPageController() {
+        try {
+            FXMLLoader loader = new FXMLLoader(getClass().getResource("/fxml_files/LoginPage.fxml"));
```
Comment (shellajt) [inline src/main/java/ledger/user_interface/ui_controllers/LoginPageController.java]: Consider pulling the page locations into a static variable. Might make future changes smoother.

File: src/main/java/ledger/database/enity/AccountBalance.java (hunk 1 above)
Comment (LadyMopar) [inline src/main/java/ledger/database/enity/AccountBalance.java]: It's still there. The diff just isn't showing it.

**Pull request merged**